doc/data/messages/u/use-dict-literal/details.rst

```diff
@@ -0,0 +1,4 @@
+https://gist.github.com/hofrob/ad143aaa84c096f42489c2520a3875f9
+
+This example script shows an 18% increase in performance when using a literal over the

```

Comment:
I really like that you went the extra mile to provide some hard facts to provide evidence! 👍 
As the last Python versions broguht some major performance improvements, it might be good to note down the Python version with which this result was obtained.